draft-ietf-ohai-svcb-config.md

```diff
@@ -238,10 +238,11 @@ for more discussion.
 
 # Gateway Location {#gateway-location}
 
-Clients that know a service is available as a target
-via discovery through the "ohttp" parameter in a SVCB or HTTPS
-record need to know the location of the associated gateway
-before sending oblivious requests.
+Once a client has discovered that a service supports Oblivious HTTP
+via the "ohttp" parameter in a SVCB or HTTPS record, it needs to be
+able to send requests via a relay to the correct gateway location.
+A relay needs to be able to learn location of the gateway associated
+with a service based on the service name alone.
```

Comment:
I don't think that this last sentence is valid.  The key consideration here is that the client needs to acquire gateway configuration that it associates with the same origin as the target.  If the relay is going to send the request to a different gateway, that is fine, as long as the gateway that is able to decrypt and respond is on the same origin as the target.

Reply:
Not so much.  You are saying that the relay cannot be given more information about the gateway than its name.  But it might be possible to provide more information than that.  There is a risk associated with that, but - to the extent that the gateway identification is usable for targeting individuals, so is its key configuration.  The problem with this statement is that it is too absolute.

Reply:
I don't see it as too absolute. I see it as the minimum amount of information the relay needs to work. It could be given more, as you say, but this text seems focused on the essential requirement.

Reply:
Yes, and that is too constraining.  Unnecessarily so.  There are just two requirements relevant to this:

1. A relay needs to know where to send requests.
2. A client needs to ensure that the information it gives the relay does not lead to it being reidentified by the gateway.

Reply:
I think (2) is handled (at least in part) by the text later in the section about not sharing the redirected paths, etc.

But we can clarify that this is not saying "the client can't share any more info with the relay" but rather "the relay needs to be able to access the gateway in case it only knows the name".

Reply:
Yes, (2) comes later.  This says that the relay needs to only be given the name, but maybe the client is the one that only gets the name and the relay is given more information by the client.